### ISSUE TITLE
The program picture is different from the original picture

### ISSUE THREAD

**Issue by ghalghay** [lovasoa/dezoomify-rs#167]
Sorry my english. Made an unpleasant discovery. It turned out that if you download the finished image [on this link](https://bibliotheques-specialisees.paris.fr/ark:/73873/pf0000738046/v0002.simple.highlight=gravure%20paris.selectedTab=thumbnail) and compare it in Photoshop with dezoomify collected — is displayed with distortions. Is it possible to do something?

Both files can be downloaded here — https://disk.yandex.ru/d/ttNuo7HDu6C7lQ

**Comment by CharlyAlonso** [comment]
Hi,
Sorry; but I don't fully understand the problem.
Do you mean what is left over on a black background?

**Comment by ghalghay** [comment]
No - I mean the image itself, not the background. If you apply one image to another - and switch the layers in Photoshop, it can be seen that the collected file contains distortion.

**Comment by ghalghay** [comment]
With accurate overlapping in contours (and not the background) - images do not coincide

**Comment by matriim** [comment]
I check and its the exact same picture only that one has cropped black background. I dont see the distortions.

**Comment by ghalghay** [comment]
See in Photoshop

**Comment by MrChrisWin** [comment]
You do not need to use dezoomify-rs for the site. There is an icon to download on top of the page. You can download to either JPEG or PDF.

**Comment by Iskander1220** [comment]
Yes, you can download it here. But it's about the distortions the program makes where you can't download.

**Comment by AlexanderDeLargeBrotherSir** [comment]
<img width="3000" height="1671" alt="Image" src="https://github.com/user-attachments/assets/d1f86c65-c0d8-4910-bc63-75e7cd6c9487" />

https://bibliotheques-specialisees.paris.fr/in/dz/68a/c04/648/HJH_RUS_CHTERENBE_001_0002n_z_files/13/{{X}}_{{Y}}.jpg

@ghalghay, you will close this right now.
